### PR TITLE
Add `History of osu! 2023`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Preface

Before talking about the actual article, I would like to address something that has been bothering me.

Writing history articles is very tedious. This article feels like a Table of Contents 2.0 for news posts. I don't know what to include other than what is already covered in news posts.

Not to mention the yearly contests that more or less are the same every time. (They are awesome but it gets boring if there is a text about them in each history article.)

I can't bring myself to finish this article for now. To not let my work go to waste, I decided to publish what I have so far for 2023. I think it's a good start.

## Notes

Okay, continuing with the article.

### Things left out on purpose

I left some stuff out that should likely reside in the news posts alone:

- The Roundtable 2
- [reese interview](https://osu.ppy.sh/home/news/2023-06-02-the-osu-youtube-legacy-of-reesewashere)
- [ephemeral interview](https://osu.ppy.sh/home/news/2023-11-10-the-followpoint-ephemeral-the-man-behind-the-medals)
- [Mapper's choice awards](https://osu.ppy.sh/home/news/2023-03-03-mappers-choice-awards)
- [osu! manga](https://osu.ppy.sh/home/news/2023-11-20-the-first-osu-manga-osu-combat-championship)
- [Halloween 2023](https://osu.ppy.sh/home/news/2023-09-24-halloween-fanart-contest) and [results](https://osu.ppy.sh/home/news/2023-10-22-halloween-descends)

### New stuff planned for addition

<details>

<summary>These are the topics that I wanted to add, sorted by month.</summary>

#### February

- [osu!monthly](https://osu.ppy.sh/home/news/2023-03-02-osumonthly-8)
  - accuracy challenge mod, multitouch tapping support for osu!, ability to remove the blur from the song select background
- osu! community meeting
  - [VOD](https://www.youtube.com/watch?v=idmI03A8jR8)
  - [notes](https://docs.google.com/document/d/13cMCrQN4vvaQFA59BmFHk6D7PH9fwT2ANmhkm5cmzxE/edit)
- [lazer update (2023-02-07)](https://www.youtube.com/watch?v=iLThnjc7Yfg)
  - multi touch input on mobile devices?

#### March

- [interview between peppy and pishifat](https://twitter.com/osugame/status/1636925240881348609)
  - problem of serving, casual, tournament players, and mappers? (due to timing constraints) (e.g. editor improvements)
- [lazer update (2023-03-01)](https://www.youtube.com/watch?v=Kmm5VlFCYE8)
  - skin editor clipboard and element depth reordering
- [lazer update (2023-03-26)](https://www.youtube.com/watch?v=5Yg9Qo5qFXc)

#### April

- Mapping contest [waifu jam!](https://osu.ppy.sh/home/news/2023-04-22-waifu-jam-map-battle-contest)
- [lazer update (2023-04-19)](https://www.youtube.com/watch?v=kmkAsXDLmNc)

#### May

- Journey into a beatmap world art contest
  - [opening post](https://osu.ppy.sh/home/news/2023-04-21-journey-into-a-beatmap-world-voting-open)
  - [results](https://osu.ppy.sh/home/news/2023-05-01-journey-into-a-beatmap-world-contest-results)
- [twin trials contest (beatmapping phase)](https://osu.ppy.sh/home/news/2023-05-15-results-twin-trials-beatmapping-phase)
- bubbles mod was added to lazer
- [lazer update (2023-05-11)](https://www.youtube.com/watch?v=GCXaQeFlaBY)

#### June

- On 6 June, ScoreV2 was implemented (#23638).[what is scorev2 (short description)] accuracy challenge mod received "maximum achievable" mode (#23317), sample bank selection added in editor (#20829), pause screen revamp (#23646, #23647)

#### July

- osu! community meeting 2nd July with peppy, smoogipoo, BTMC, Walavouchey and others from the community
  - [VOD](https://www.youtube.com/watch?v=bocgaVISSfg)
  - [newspost recap](https://osu.ppy.sh/home/news/2023-07-11-community-meeting-recap)
- [new beatmap pack categories](https://osu.ppy.sh/home/news/2023-07-11-introducing-new-beatmap-pack-categories)
- [kudosu ranking page](https://osu.ppy.sh/p/kudosu) was [reimplemented on new website](https://github.com/ppy/osu-web/pull/10344)
- [lazer update (2023-07-16)](https://osu.ppy.sh/home/news/2023-07-16-osulazer-updates-july-16-2023)
  - score recalculation to ScoreV2
  - more search options at song select
- [community meeting (2023-07-07)](https://osu.ppy.sh/home/news/2023-07-11-community-meeting-recap)
- beach episode art contest
  - [opening post](https://osu.ppy.sh/home/news/2023-06-20-beach-episode-art-contest)
  - [results](https://osu.ppy.sh/home/news/2023-07-27-summer-splash)
  - peppy's picks?
  - 8 new hush-hush medals -> include mention about UberFazz winning the internment medal and the medal hunter title?

#### August

- [lazer update (2023-08-11)](https://osu.ppy.sh/home/news/2023-08-11-osulazer-updates-august-11-2023)
  - kiai fountains
  - Tournament client improvements
- [osu! Malaysian event](https://osu.ppy.sh/home/news/2023-08-22-osu-malaysia-animangaki)
- [community meeting (2023-08-21)](https://www.youtube.com/watch?v=O2j00E6AVW8): discussing the missing pieces to lazer ranked play

#### September

- [osu!(lazer) update (2023-09-08)](https://osu.ppy.sh/home/news/2023-09-08-osulazer-updates-the-path-to-ranked-play)
  - ranked play: GitHub project board; discussion with a handful of top players
  - Sorting by difficulty splits difficulties into individual panels

#### October

- [osu!(lazer) Updates: October 8, 2023](https://osu.ppy.sh/home/news/2023-10-08-osulazer-updates-october-8)
  - argon skin changes
  - Extended information on mod icons
  - Lower maximum required spinner RPM cap based on Overall Difficulty
- [osu!(lazer) Updates: October 26, 2023](https://osu.ppy.sh/home/news/2023-10-26-osulazer-updates-october-26)
  - Player invites for multiplayer rooms

#### November

- [osu!(lazer) Updates: November 14, 2023](https://osu.ppy.sh/home/news/2023-11-14-osulazer-updates-november-14)
  - new argon design
  - new main menu sound effects
- [osu!(lazer) Updates: November 30, 2023](https://osu.ppy.sh/home/news/2023-11-30-osulazer-updates-november-30)
  - Free-hand slider drawing
  - HP drain rate now matches osu!(stable)
  - skin editor accessible from the main menu

</details>

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by TicClick — DISMISSED** [review]
honestly it's an awesome work, and I regret having not read it sooner

File: wiki/History_of_osu!/2023/en.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+Thanks to [ThePoon](https://osu.ppy.sh/users/718454) and other contributors, [osu!stream](/wiki/osu!stream) was ported to [Android](https://en.wikipedia.org/wiki/Android_(operating_system)), a game specifically designed for handheld devices and with unique gameplay elements. Until then, it was only available on [iOS](https://en.wikipedia.org/wiki/IOS). An implementation on Android was not planned at the time when the development of the iOS version began in 2011 due to audio-related restrictions in Android's infrastructure that would heavily affect the gameplay experience.[^osu-stream-twitter][^osu-stream-pr]
```
Comment (iSlodinxOsu) [inline wiki/History_of_osu!/2023/en.md:9]:
```suggestion
Thanks to [ThePooN](https://osu.ppy.sh/users/718454) and other contributors, [osu!stream](/wiki/osu!stream) was ported to [Android](https://en.wikipedia.org/wiki/Android_(operating_system)), a game specifically designed for handheld devices and with unique gameplay elements. Until then, it was only available on [iOS](https://en.wikipedia.org/wiki/IOS). An implementation on Android was not planned at the time when the development of the iOS version began in 2011 due to audio-related restrictions in Android's infrastructure that would heavily affect the gameplay experience.[^osu-stream-twitter][^osu-stream-pr]
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/2023/en.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+[Dynam1cNET](https://osu.ppy.sh/users/12540789) wanted to show what is possible with the me! page and implemented a custom version of r/place, an event on [Reddit](https://www.reddit.com) where users were able to *place* coloured pixels on a canvas.[^me-page-place]
+
+## November
+
+Several people were commended and awarded with a Community Contributor badge for their remarkable assistance in the osu! community in 2022 and 2023.[^contributors]
+
+## December
+
+In parallel to this year's last update, the osu! team wanted more people to try out osu!(lazer) and give feedback via the forum before locking gameplay mechanics and adding the pp system.[^feedback] Gaining a miss caused less penalty for high combos (`combo^1.5` compared to `combo^2`). The combo–accuracy split was rebalanced, making accuracy account for a bigger portion in a score. osu!catch's grading in osu!(lazer) was changed to match the grading of osu!(stable) to simplify obtaining high accuracy values. At last, sliders became more lenient.[^lazer-12-18]
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2023/en.md:65]:
```suggestion
In parallel to this year's last update, the osu! team wanted more people to try out osu!(lazer) and give feedback via the forum before finalising gameplay mechanics and allowing [performance points](/wiki/Performance_points) to be awarded for scores.[^feedback] The following notable changes happened:

- The score penalty for misses at high combos was decreased (`combo^1.5` compared to `combo^2`).
- The combo–accuracy split was rebalanced, making accuracy account for a bigger portion in a score.
- osu!catch's grading in osu!(lazer) was changed to match the grading of osu!(stable) to simplify obtaining high accuracy values.
- Slider mechanics were adjusted to make fast sliders more lenient.[^lazer-12-18]
```

File: wiki/History_of_osu!/2023/en.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+The 2024's [Winter Fanart Contest](https://osu.ppy.sh/home/news/2023-11-07-winter-fanart-contest) ran under the theme "osu! mascots enjoying a seasonal feast". Due to this broad theme, the participants had a lot of freedom and creativity for designs. All finalists obtained three months of osu!supporter, while the top 3 winners received a custom [profile badge](/wiki/Community/Profile_badge) on top.[^winter-fanart]
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2023/en.md:67]:
```suggestion
2024's [Winter Fanart Contest](https://osu.ppy.sh/home/news/2023-11-07-winter-fanart-contest) ran under the theme "osu! mascots enjoying a seasonal feast". Due to this broad theme, the participants had a lot of freedom and creativity for designs. All finalists obtained three months of osu!supporter, while the top 3 winners received a custom [profile badge](/wiki/Community/Profile_badge) on top.[^winter-fanart]
```

File: wiki/History_of_osu!/2023/en.md (hunk 1 above)
Comment (iSlodinxOsu) [reply wiki/History_of_osu!/2023/en.md:9]: Yeah nvm u can ignore this, Wala did some changes

[the remark this answers is not in the document]
Comment (iSlodinxOsu) [reply wiki/History_of_osu!/2023/en.md:9]: Should be capitalized with `ThePooN` btw

**Comment by The-Last-Cookie** [comment]
I will add the image and some local commits later.

**Review by Walavouchey — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**